src/templates/specimen/body/annotate/annotationTypes/RemoveBlock.js

```diff
@@ -0,0 +1,30 @@
+import { Row, Col } from 'react-bootstrap';
+
+
+const RemoveBlock = (props) => {
```

Comment:
Is this the right name, it seems to also save

Reply:
You're right, changed name of component to 'ActionsBlock'